assets/scripts/streets/width.js

```diff
@@ -227,8 +60,8 @@ export function recalculateOccupiedWidth () {
     street.remainingWidth = 0
   }
 
-  buildStreetWidthMenu()
-  updateStreetMetadata(street)
+  window.dispatchEvent(new CustomEvent('stmx:width_updated'))
+  // updateStreetMetadata(street)
```

Comment:
Various areas where `updateStreetMetadata` is commented out - how certain are we this isn't needed anymore and we can just remove those lines? i haven't noticed anything wrong yet